process/templates/PROJECT_NAME_incubation_stage.md

```diff
@@ -1,5 +1,8 @@
 ## Project incubation application
 
+### Project has met all Sandbox requirement
+  * "link to sandbox PR"
```

Comment:
Same comment as before: I think there is no need to ask people to dig this up.

Reply:
Is it acceptable to leave in the form, but grandfather in existing projects to NOT have to do this, but all NEW work would need to provide the link as they progress?

Reply:
added in